UsbIpServer/WslDistributions.cs

```diff
@@ -85,57 +87,58 @@ public static async Task<WslDistributions> CreateAsync(CancellationToken cancell
             //   Debian             Stopped         2
             //   Custom-MyDistro    Running         2
             var detailsResult = await ProcessUtils.RunCapturedProcessAsync(WslPath, new[] { "--list", "--all", "--verbose" }, Encoding.Unicode, cancellationToken);
-            if (detailsResult.ExitCode != 0)
+            // This fails with non-zero exit code (and some help text) if there are no distributions installed.
+            // We have already checked 'wsl --status', so WSL itself *is* installed.
+            // Therefore, we just treat this as an empty distro-list, which is actually what it is.
+            if (detailsResult.ExitCode == 0)
```

Comment:
This is the actual change, all the rest is simply from indenting the code inside the `if`